src/config_file.rs

```diff
@@ -0,0 +1,46 @@
+///! This module provides methods to handle the program's config files and operations related to
+///this.
+use std::fs::File;
+use std::io::prelude::*;
+
+use xdg::BaseDirectories;
+use yaml_rust::{Yaml, YamlLoader};
+
+const CONF_DIR: &str = "lsdeluxe";
```

Comment:
May this should be `lsd` instead of `lsdeluxe`?

Reply:
I had an internal debate about that with myself, yeah. Sure `lsd` is closer to the executable name, but keep in my, that you are sharing these directories with an unknown number of other applications. That's why I thought being a bit more verbose here avoids any naming conflicts in the future. We would just need to make sure users know, that `lsd` stands for `lsdeluxe`. But I can also shorten it, if you guys think it being recognizable outweighs the risk of naming conflicts.

Reply:
I was thinking we use `lsd` for a two main reasons:

- I have not really seen any other packages called `lsd`.
- I don't know how many people know `lsd` stands for `lsdeluxe`

---

Do you have any other packages in mind that will clash with `lsd`?

Reply:
I think it would be a good idea to keep the name of the binary and config folder same.

Reply:
I renamed it now.